packages/nextjs/app/builders/0x04cdfb463600a29e8e93053b88daa11f54379907/page.tsx

```diff
@@ -0,0 +1,467 @@
+            </div>
+          </div>
+
+          <div className="flex flex-col items-center text-center space-y-8 w-full p-8 md:p-12">
+            <div className="space-y-5 flex flex-col items-center">
+              <div className="relative group">
+                <div className="absolute -inset-1 bg-gradient-to-tr from-red-600 via-rose-500 to-orange-500 rounded-full blur opacity-60 group-hover:opacity-100 transition duration-500"></div>
+                <div className="relative w-32 h-32 rounded-full p-1 bg-gradient-to-tr from-red-500 to-orange-500 shadow-2xl flex items-center justify-center">
+                  <div
+                    className={`w-full h-full rounded-full border-4 flex items-center justify-center overflow-hidden ${isDark ? "border-[#0f172a] bg-slate-800" : "border-slate-400 bg-[#e2e8f0]"}`}
+                  >
+                    <Image
+                      src="/var.png"
+                      alt="Kohen Eric Avatar"
+                      width={120}
+                      height={120}
+                      className="rounded-full object-cover"
+                    />
+                  </div>
+                </div>
+              </div>
+
+              <div>
+                <h1
+                  className={`text-4xl md:text-6xl font-black tracking-tight mb-2 ${isDark ? "text-white" : "text-slate-900"}`}
+                >
+                  Kohen Eric
+                </h1>
+                <p className={`${theme.mutedText} text-sm uppercase tracking-[0.2em] font-bold`}>Batch 22 Graduate</p>
+              </div>
+
+              <p className={`${theme.subText} max-w-lg leading-relaxed text-sm md:text-base font-medium opacity-90`}>
+                Smart contract developer building on Arbitrum. Passionate about creating secure Web3 infrastructure.
+              </p>
+
+              <button
+                onClick={handleCopy}
+                className={`group flex items-center gap-3 transition-all rounded-full py-2 pl-2 pr-5 max-w-full cursor-pointer backdrop-blur-sm border ${theme.walletBg}`}
+              >
+                <div className="w-8 h-8 rounded-full bg-gradient-to-br from-red-500 to-orange-500 flex items-center justify-center text-[10px] font-bold text-white shadow-inner flex-shrink-0">
+                  0x
+                </div>
+                <span
+                  className={`font-mono text-sm truncate group-hover:text-red-500 transition-colors ${theme.walletText}`}
+                >
+                  {WALLET_SHORT}
+                </span>
+                <div className="flex-shrink-0">
+                  {copied ? (
+                    <Check size={16} className="text-green-500" />
+                  ) : (
+                    <Copy
+                      size={16}
+                      className={`${isDark ? "text-slate-500" : "text-slate-600"} group-hover:text-red-500`}
+                    />
+                  )}
+                </div>
+              </button>
```

Comment:
As I said, we are using Scaffold-UI on this project, and there's a component that renders the address. You should try using it. Here are the docs: [https://scaffold-ui-docs.vercel.app/](https://scaffold-ui-docs.vercel.app/)